src/isa/riscv64/instr/rvv/decode.h

```diff
@@ -321,8 +298,8 @@ def_THelper(vopm) {
   def_INSTR_TAB("001111 ? ????? ????? ??? ????? ????? ??", vslide1down);
   
   def_INSTR_TAB("010000 ? ????? ????? ??? ????? ????? ??", vrwxunary0_dispatch);
-  def_INSTR_TAB("010010 ? ????? ????? ??? ????? ????? ??", vxunary0_dispatch);
-  def_INSTR_TAB("010100 ? ????? ????? ??? ????? ????? ??", vmunary0_dispatch);
+  def_INSTR_TAB("010010 ? ????? ????? 010 ????? ????? ??", vxunary0_dispatch);
+  def_INSTR_TAB("010100 ? ????? ????? 010 ????? ????? ??", vmunary0_dispatch);
```

Comment:
There is no need to modify this field, since it has been checked in its parent THepler

Reply:
complete